src/web/web-service/wpan_service.hpp

```diff
@@ -146,7 +147,7 @@ class WpanService
      * @param[in]  aIfName  The pointer to the interface name of wpantund.
      *
      */
-    void SetInterfaceName(const char *aIfName) { strncpy(mIfName, aIfName, sizeof(mIfName)); }
+    void SetInterfaceName(const char *aIfName) { strncpy(mIfName, aIfName, sizeof(mIfName) - 1); }
```

Comment:
http://www.cplusplus.com/reference/cstring/strncpy/
> No null-character is implicitly appended at the end of destination if source is longer than num

We need to manually set `\0` at the end of `mIfName`.

Reply:
fixed by using strcpy_safe